tools/pom.xml

```diff
@@ -22,7 +22,7 @@
 		<maven.compiler.target>11</maven.compiler.target>
 		<!-- Note: see https://github.com/unicode-org/icu/packages/411079/versions
 			for the icu4j.version tag to use -->
-		<icu4j.version>72.0.1-SNAPSHOT-cldr-2022-08-11</icu4j.version>
```

Comment:
this is an unexpected change.

Reply:
yes, surprising that it's combined with the plugin change -- and does dependabot actually know about icu4j snapshot versions?

Reply:
I figured it out just now. It's actually this way (08-17) in `main`.  So dependabot did this change to avoid a merge conflict.

Reply:
if it already matches main, though, why does it show up in the diff?